packages/gatsby-cli/src/init-starter.ts

```diff
@@ -341,8 +341,24 @@ export async function initStarter(
   trackCli(`NEW_PROJECT`, {
     starterName: hostedInfo ? hostedInfo.shortcut() : `local:starter`,
   })
-  if (hostedInfo) await clone(hostedInfo, rootPath)
-  else await copy(starterPath, rootPath)
+  if (hostedInfo) {
+    await clone(hostedInfo, rootPath)
+  } else {
+    await copy(starterPath, rootPath)
+  }
+
+  const sitePath = sysPath.resolve(rootPath)
+
+  const sitePackageJson = await fs
+    .readJSON(sysPath.join(sitePath, `package.json`))
+    .catch(() => {})
```

Comment:
Nit: Add a verbose log instead of the infamous swallow catch `() => {}`

Reply:
I slap my own hand! I was complaining about exactly this last week!

Reply:
I know 😉

Reply:
Fixed